spec/index.bs

```diff
@@ -2733,6 +2757,79 @@ Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUTextureCopyView}}
 
 </div>
 
+### <dfn method for=GPUCommandEncoder>copyTextureToTexture(source, destination, copySize)</dfn> ### {#GPUCommandEncoder-copyTextureToTexture}
+
+<div algorithm="GPUCommandEncoder.copyTextureToTexture">
+
+  **Arguments:**
+    - {{GPUTextureCopyView}} |source|
+    - {{GPUTextureCopyView}} |destination|
+    - {{GPUExtent3D}} |copySize|
+
+  **Returns:** void
+
+  Encode a command into the {{GPUCommandEncoder}} that copies data from a sub-region of one
+  or multiple continuous {{GPUTexture}} [=subresources=] to another sub-region of one or
```

Comment:
nit: Should use "contiguous" instead of "continuous", per mathematics convention, because this describes neighbors in a discrete set, rather than a continuous range (`[1,2,3]` is a contiguous range of integers, `[1.0, 3.0)` is a continuous range of real numbers).